keyboards/cipulot/ec_pro2/keymaps/via/via_apc.c

```diff
@@ -0,0 +1,156 @@
+/* Copyright 2023 Cipulot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#    include "ec_switch_matrix.h"
+#    include "quantum.h"
```

Comment:
```suggestion
#include "ec_switch_matrix.h"
#include "eeconfig.h"
````

Reply:
Actually, this causes errors at compile time with the `via_command_ids`. 
Adding `#include "eeconfig.h"` fixes the via ids but prompts unknown type names for keycodes in via.h compile. Reverting back to `quantum.h` to fix these errors.

Reply:
quantum.h is overkill here, you should only include the headers that provide symbols you're using.

Reply:
<img width="1064" alt="Screenshot 2023-02-23 at 4 08 39 PM" src="https://user-images.githubusercontent.com/40441626/220946527-48c4fc15-1964-44ab-8054-24210f5e3a79.png">

Indeed that's overkill. Including the following gives the error in the picture.

Reply:
This shouldn't be an issue in a few days, the keycodes have been removed from via.h in develop.

Reply:
I pushed a fix for that error. If it needs to be revised when the new merge happens I'll handle it